### ISSUE TITLE
Conversion can overflow ignores assignment when there is a division present.

### ISSUE THREAD

**Issue by movedoa** [Riverside-Software/sonar-openedge#930]
The last one doesn't get recognized as an error.

```
DEFINE VARIABLE liFoo AS INT64 NO-UNDO.

/* Error */
liFoo = INTEGER(liTime).

/* Error */
liFoo = INTEGER(liTime * 1).

/* Error */
liFoo = INTEGER(liTime - 1).

/* Error */
liFoo = INTEGER(liTime + 1).

/* No Error */
liFoo = INTEGER(liTime / 1).
```

**Comment by gquerret** [comment]
Confirmed, cast from `DECIMAL` to `INTEGER` is currently not caught by the rule

**Comment by gquerret** [comment]
Conversion from DECIMAL to INT now handled correctly. Another case will probably be added (in this rule) to report the loss of the decimal value when converting from DECIMAL to INT[64]
